defactor-com/ui-kit
# Feat/implement empty state to charts 152

## Implement empty state to charts

### What does this PR do?

- Resolve #152 

### Steps to test

1. Run project
2. Check changes

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/3d258294-5ee2-4187-a8fd-e126b66eabb8)


#### CheckList
